bamboo-specs/bamboo.yaml

```diff
@@ -276,13 +292,89 @@ Release ORCA Code:
         set -ex
         bin/build_release_package.sh
       description: Create Release Files and Release in GitHub
+
+#job for deploying cumulus and orca
+Deploy RDS cluster:
+  key: DRC
+  other:
+    clean-working-dir: true
+  docker:
+    image: maven.earthdata.nasa.gov/cumulus_orca
+    volumes:
+      ${bamboo.working.directory}: ${bamboo.working.directory}
+      ${bamboo.tmp.directory}: ${bamboo.tmp.directory}
+  tasks:
+  - checkout:
+      force-clean-build: 'true'
+      description: Checkout Default Repository
+  - script:
+      interpreter: SHELL
+      scripts:
+      - |-
```

Comment:
What is this?

Reply:
added comment to the first two places it was used.